### PR TITLE
added CourseComparison

### PR DESCRIPTION
**Pull request by Shembou**

### CODE REVIEW

File: sanity/schemas/components/CourseComparison.js [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+      validation: (Rule) => Rule.required(),
+    },
+  ],
+  preview: {
+    select: {
+      heading: 'heading',
+    },
+    prepare({heading}) {
+      return {
+        title: `[${title}] - ${removeMarkdown(heading)}`,
```
Comment (milewskibogumil) [inline sanity/schemas/components/CourseComparison.js:39]: add subtitle of paragraph.

File: sanity/schemas/components/CourseComparison.js [hunk 2]
```diff
@@ -0,0 +1,80 @@
+import {removeMarkdown} from '../../utils/remove-markdown'
+
+const icon = () => '📊'
+const title = 'Porównanie kursów'
+
+export default {
+  title,
+  name: 'CourseComparison',
+  type: 'object',
+  icon,
```
Comment (milewskibogumil) [inline sanity/schemas/components/CourseComparison.js]: Ordering (change everywhere):
1. name
2. type
3. title
4. icon

File: sanity/schemas/components/CourseComparison.js [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+      of: [{type: 'string'}],
+      title: 'Lista',
+      validation: (Rule) => Rule.required(),
+    },
+    {
+      name: 'positive',
+      type: 'boolean',
+      title: 'Czy jest pozytywny?',
+      initialValue: true,
+    },
```
Comment (milewskibogumil) [inline sanity/schemas/components/CourseComparison.js]: Here's a wring schema. `positive` should be inside list. List should have types
```
list: {
  name: string;
  isPositive: boolean;
}[];
```

**Review by milewskibogumil — CHANGES_REQUESTED** [review]

File: sanity/schemas/components/CourseComparison.js (hunk 3 above)
Comment (Shembou) [reply sanity/schemas/components/CourseComparison.js]: are you sure it is the correct approach? Please think about it for a second.

**Pull request merged**